CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# Specify the minimum version.
+cmake_minimum_required ( VERSION 3.0 )
+
+# Specify the project info.
+project ( trixi VERSION 0.1.0 DESCRIPTION "Interface library for using Trixi.jl from C/C++/Fortran" )
+
+# Enable fortran
+enable_language(Fortran)
```

Comment:
Do we also need to enable C? Or, for clarity, should we just include it even if it is enabled automatically to make it explicit?

Reply:
I will add it.